Simulator/Test/test_matrix_math.py

```diff
@@ -9,17 +10,199 @@
 
 import sun_sensor_math as LA
 
+def pytest_addoption(parser):
+    parser.addoption(
+        "--cmdopt", action="store", default="type1", help="my option: type1 or type2"
+    )
+
+@pytest.fixture
+def cmdopt(request):
+    return request.config.getoption("--cmdopt")
+
 # usual convention is to name the function "test_<function_to_test>"
-def test_matTimesVec():
+def test_test_vecTimesMat():
```

Comment:
why is the name `test_test...`? Is there a reason?

Reply:
nope, I'll fix that